pkg/bypass4netns/socket.go

```diff
@@ -301,6 +307,9 @@ func (ss *socketStatus) handleSysConnect(handler *notifHandler, ctx *context) {
 }
 
 func (ss *socketStatus) handleSysBind(pid int, handler *notifHandler, ctx *context) {
+	if ss.ignoreBind {
+		return
```

Comment:
It would be better to disable bypass for the socket with `ss.state = NotBypassable`

Reply:
Fixed